hrms/hr/doctype/employee_advance/employee_advance.py

```diff
@@ -109,7 +109,7 @@ def set_total_advance_paid(self):
 				EmployeeAdvanceOverPayment,
 			)
 
-		if flt(return_amount) > self.paid_amount - self.claimed_amount:
+		if flt(return_amount) > round(self.paid_amount - self.claimed_amount, 2):
```

Comment:
@kittiu Thanks for attempting a fix, but directly rounding to 2 digits is incorrect. 

Will have to keep the precision same throughout the calculation and respect system precision too. Will raise a separate fix for this

Reply:
What about setting precision = 2 in the amount fields?
Note: I would love to learn how do you normally cope with these decimal diff too, as I see these diff many many places.

Reply:
> What about setting precision = 2 in the amount fields?

Precision can be customized/overridden at the field and system level, so this won't help.
https://docs.erpnext.com/docs/v14/user/manual/en/setting-up/articles/set-precision

Field's precision will have to be fetched and all calculations should respect it

Reply:
In this case, meaning, the claimed amount should be fetched = 1730.404, but then, I want to return 269.60, this is throw error.
Because
`flt(return_amount) > self.paid_amount - self.claimed_amount `
`269.60 > (2,000 - 1730.404)`
`269.60 > 269.596` is true
I could help fix too if you have sample somehwere :)